src/rtctools/optimization/csv_mixin.py

```diff
@@ -295,3 +279,19 @@ def write_output(ensemble_member, folder):
                 )
         else:
             write_output(0, self._output_folder)
+
+
+def get_timeseries_from_csv(
+    file: Path, delimiter=","
+) -> Tuple[List[datetime], Dict[str, np.ndarray]]:
+    """Read timeseries from csv
+
+    :param file:        Path to CSV.
+    :param delimiter:   CSV column delimiter.
+
+    :returns: A list of time stamps (datetimes) and dict of values.
+    """
+    data = csv.load(file, delimiter=delimiter, with_time=True)
+    times = data[data.dtype.names[0]]
+    var_dict = {key: np.asarray(data[key], dtype=np.float64) for key in data.dtype.names[1:]}
+    return times, var_dict
```

Comment:
The refactored 'MultiSeedMixin' does not require any of the changes to 'PIMixin' and 'CSVMixin'. Should the latter changes be removed or moved to a separate PR?

Reply:
Yes, I can make a separate PR for that.